lib/Graylog.js

```diff
@@ -0,0 +1,427 @@
+var zlib = require('zlib');
+var crypto = require('crypto');
+var dgram = require('dgram');
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+var Queue = require('./Queue');
+
+var MAX_SAFE_INT = 9007199254740991;  // Number.MAX_SAFE_INTEGER
```

Comment:
JavaScript versions. Although since we now depend on a newer Node, this may indeed be overkill.